.changes/unreleased/FEATURES-20230213-213454.yaml

```
@@ -0,0 +1,6 @@
+kind: FEATURES
+body: 'resource/random_bytes: New resource that generates an array of random bytes
+  intended to be used as key or secret ([#63](https://github.com/hashicorp/terraform-provider-random/issues/63)).'
```

```suggestion
  intended to be used as key or secret'
```

Apologies, believe I suggested including a link in the changie file. This is not needed as changie will generate a link on the basis of the specified `Issues`.